Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertSpace.cs

```diff
@@ -0,0 +1,117 @@
+﻿using Autodesk.Revit.DB;
+using Objects.BuiltElements;
+using Objects.Geometry;
+using Speckle.Core.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+using DB = Autodesk.Revit.DB.Mechanical;
+using Point = Objects.Geometry.Point;
+
+namespace Objects.Converter.Revit
+{
+    public partial class ConverterRevit
+    {
+        public List<ApplicationPlaceholderObject> SpaceToNative(Space speckleSpace)
+        {
+            var revitSpace = GetExistingElementByApplicationId(speckleSpace.applicationId) as DB.Space;
+            var level = LevelToNative(speckleSpace.level);
+            var basePoint = PointToNative(speckleSpace.basePoint);
+            var upperLimit = LevelToNative(speckleSpace.upperLimit);
+
+            // create new space if none existing, include zone information if available
+            if (revitSpace == null)
+            {                
+                revitSpace = Doc.Create.NewSpace(level, new UV(basePoint.X, basePoint.Y));
+                if (speckleSpace.zoneId != null)
+                {
+                    var speckleZone = Doc.GetElement(speckleSpace.zoneId) as DB.Zone;
```

Comment:
(see commend below on element ids)

Reply:
Noted, updated to using element name of zone, as opposed to element id, as recommended.